### DISCUSSION TITLE
Fitting with Prisma

### DISCUSSION THREAD

**Discussion by MentalGear** [noam-honig/radweb#65] [Q&A]
I'm wondering how this would fit with prisma ORM?

**Comment by yoni-rapoport** [comment]
Remult comes with its own type-safe database client that uses model classes (not code-generated) as the source of truth for db queries. This is a very different approach compared to prisma.

We've had cases where people started out with prisma and then switched over to Remult but kept using the database schema created with prisma. (This may require some data-type configs).

Also in back-end code you can keep using prisma (e.g. non-Remult routes, Remult [BackendMethods](https://remult.dev/docs/backendMethods.html), etc...).
